README.md

```diff
@@ -64,8 +65,19 @@ A Rails app with `ActiveJob` configured.
 
 ## Installation
 
+Before you can send data to BigQuery with `dfe-analytics` you'll need to setup
+your Google Cloud project. See the [setup Google Cloud setup guide](docs/GOOGLE_CLOUD_BIGQUERY_SETUP.md)
```

Comment:
```suggestion
your Google Cloud project. See the [setup Google Cloud setup guide](docs/google-cloud-bigquery-setup.md)
```

Feels a little odd to have the filename all in caps, but not a big deal.

Reply:
Ya, agreed, will change.